packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -147,7 +147,7 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
     return (
       <div className={ textFieldClassName }>
         { label && <Label htmlFor={ this._id }>{ label }</Label> }
-        <div className={ css(styles.fieldGroup) }>
+        <div className={ css(styles.fieldGroup, isFocused && styles.fieldGroupIsFocused) }>
```

Comment:
@cschleiden i believe we're actually trying to move away from that notation for compatibility with Glamor.